swift-6-beta.docc/LanguageGuide/OpaqueTypes.md

```diff
@@ -1,41 +1,16 @@
-<!--
-要翻译的文件：https://github.com/SwiftGGTeam/the-swift-programming-language-in-chinese/blob/swift-6-beta-translation/swift-6-beta.docc/LanguageGuide/OpaqueTypes.md
-Swift 文档源文件地址：https://docs.swift.org/swift-book/documentation/the-swift-programming-language/opaquetypes
-翻译估计用时：⭐️⭐️⭐️⭐️⭐️
--->
-
-# Opaque and Boxed Protocol Types
+# 不透明类型和封装协议类型
 
-Hide implementation details about a value's type.
+隐藏关于值类型的实现细节。
 
-Swift provides two ways to hide details about a value's type:
-opaque types and boxed protocol types.
-Hiding type information
-is useful at boundaries between
-a module and code that calls into the module,
-because the underlying type of the return value can remain private.
+Swift 提供了两种隐藏值类型细节的方法：不透明类型（Opaque Type）和封装协议类型（Boxed Protocol Type）。在模块与模块调用端之间的边界上隐藏类型信息是有用的，因为这样返回值的底层类型可以保持为私有的可访问状态。
```

Comment:
> Hiding type information is useful at boundaries between a module and code that calls into the module, because the underlying type of the return value can remain private.

建议翻译为，在隔离模块和调用模块的代码上，隐藏类型信息是很有用处的，因为返回值的底层类型仍然保持私有。